experiments/golden-results/StratoX-summary.txt

```diff
@@ -59,15 +59,15 @@ Error message: Unsupported pragma: Check
 Nkind: N_Pragma
 --
 Occurs: 8 times
-Calling function: Process_Pragma_Declaration
-Error message: Unsupported pragma: Obsolescent
-Nkind: N_Pragma
---
-Occurs: 7 times
 Calling function: Do_Type_Reference
 Error message: Type of type not a type
 Nkind: N_Defining_Identifier
 --
+Occurs: 8 times
+Calling function: Process_Pragma_Declaration
+Error message: Unsupported pragma: Obsolescent
+Nkind: N_Pragma
```

Comment:
Most of the golden results changes appear to be just things moving around for some weird reason. Very few actual changes.

Reply:
That looks like a move, but actually its because "Do_Type_Reference" went fomr 7 -> 8 times...

Reply:
@chrisr-diffblue Yes, I believe this is because some of the nodes that we now handle were not handled before. This means that some nodes in these projects now propagate, and they might result into some previously unreported errors showing up.

Reply:
@chrisr-diffblue Keep in mind that the changes made in this PR cause gnat2goto to not crash in some circumstances where it would’ve crashed before.